calligraphy/build.gradle

```diff
@@ -21,6 +21,7 @@ android {
 }
 
 dependencies {
+    compile 'io.github.inflationx:viewpump:0.1.0-SNAPSHOT'
     compile 'com.android.support:appcompat-v7:24.0.0'
```

Comment:
@chrisjenx what do you think about dependencies here? keep viewpump compile or change to provided?  what about appcompat, too?